packages/caver-klay/caver-klay-kct/contract/token/KIP7/IKIP7Receiver.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @title KIP-7 Fungible Token Standard, optional wallet interface
+ * @dev Note: the KIP-13 identifier for this interface is 0x9d188c22.
+ * see http://kips.klaytn.com/KIPs/kip-7-fungible_token
+ */
+contract IKIP7Receiver {
+    /**
+     * @notice Handle the receipt of KIP-7 token
+     * @dev The KIP-7 smart contract calls this function on the recipient
+     *  after a `transfer`. This function MAY throw to revert and reject the
```

Comment:
I think `safeTransfer` is right. So I modified this and KIP-17 also in KIP.
In openzeppline, `IERC721Receiver`'s description is also same.

```suggestion
     *  after a `safeTransfer`. This function MAY throw to revert and reject the
```